csrankings-7.csv

```diff
@@ -263,6 +263,7 @@ Qi Han 0001,Colorado School of Mines,http://inside.mines.edu/~qhan,CiPyIk4AAAAJ
 Qi Li 0002,Tsinghua University,https://www.cs.tsinghua.edu.cn/publish/csen/4623/2020/20200105201627634466124/20200105201627634466124_.html,NOSCHOLARPAGE
 Qi Li 0012,Iowa State University,https://www.cs.iastate.edu/people/qi-li,NOSCHOLARPAGE
 Qi Liu 0003,USTC,http://staff.ustc.edu.cn/~qiliuql,5EoHAFwAAAAJ
+Qi Sun,New York University,http://qisun.me/,oN7gaqMAAAAJ

```

Comment:
The current DBLP entry is not a proper page - please contact DBLP per the instructions to get a proper unambiguous page. (See https://dblp.org/pid/05/4187.html)